frame/utility/src/lib.rs

```diff
@@ -230,7 +231,7 @@ decl_module! {
 			|args: (&Vec<<T as Trait>::Call>,)| {
 				args.0.iter()
 					.map(|call| call.get_dispatch_info().weight)
-					.fold(10_000, |a, n| a + n)
+					.fold(65_000_000 as Weight, |a, n| a.saturating_add(n))
```

Comment:
I think `as` should be avoided when possible as it silently truncates instead of throwing a compilation error. I am not sure if you even need it here as type inference should inference the correct literal. If not possible to omit I would rather write:
```
.fold(65_000_000, |a: Weight, n| a.saturating_add(n))
```